  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  joint1_position_controller:
    type: effort_controllers/JointPositionController
    joint: arm_base_joint
    pid: {p: 1.0, i: 1.0, d: 0.0}

  joint2_position_controller:
    type: effort_controllers/JointPositionController
    joint: base_to_arm_1_joint
    pid: {p: 2.0, i: 1.0, d: 0.0}

  joint3_position_controller:
    type: effort_controllers/JointPositionController
    joint: arm_1_to_arm_2_joint
    pid: {p: 2.0, i: 1.0, d: 0.0}

  joint4_position_controller:
    type: effort_controllers/JointPositionController
    joint: arm_2_to_arm_3_joint
    pid: {p: 2.0, i: 1.0, d: 0.0}





